pkg/server/handlers.go

```diff
@@ -63,6 +65,11 @@ func (s *Server) handlePropertyValueHistoryQuery(ctx context.Context, req *backe
 		return DataResponseErrorRequestFailed(err)
 	}
 
+	wide, err := data.LongToWide(frames[0], &data.FillMissing{Mode: data.FillModeNull, Value: math.NaN()})
```

Comment:
this looks great -- but would love to see the test structure change to show we now return a wide frame